.github/workflows/build-plugin-archive.yml

```diff
@@ -98,6 +103,20 @@ jobs:
               echo "archive-name=${{ github.event.repository.name }}" >> $GITHUB_OUTPUT
           fi
 
+      - name: Set up plugin dir name
+        env:
+          PLUGIN_DIR_NAME: ${{ inputs.PLUGIN_DIR_NAME }}
+          ARCHIVE_NAME: ${{ inputs.ARCHIVE_NAME }}
+        id: plugin-dir-name
+        run: |
+          if [ ! -z "$PLUGIN_DIR_NAME" ]; then
+            echo "plugin-dir-name=$PLUGIN_DIR_NAME" >> $GITHUB_OUTPUT
+          elif [ ! -z "$ARCHIVE_NAME" ]; then
+            echo "plugin-dir-name=$ARCHIVE_NAME" >> $GITHUB_OUTPUT
+          else
+            echo "plugin-dir-name=${{ github.event.repository.name }}" >> $GITHUB_OUTPUT
+          fi
```

Comment:
ChatGPT [says](https://chat.openai.com/share/80452a8e-b1dc-43e1-aba1-5920e8c8d9dd) this could be simplified:
```suggestion
        id: plugin-folder-name
        run: |
          echo "plugin-folder-name=${PLUGIN_FOLDER_NAME:-$ARCHIVE_NAME:-${{ github.event.repository.name }}}" >> $GITHUB_OUTPUT
```
(untested!)

Reply:
Oh, wow, that's cool :)

I'm pretty sure this will work, but let's test it.